doc/src/base/io-network.md

```diff
@@ -6,6 +6,8 @@
 Base.stdout
 Base.stderr
 Base.stdin
+Base.read(::String)
+Base.write(::String, ::Any)
```

Comment:
```suggestion
Base.read(::AbstractString)
Base.write(::AbstractString, ::Any)
```

Should we just move all of the content here though? I feel like duplicating the content might just be a bit more confusing for search.